### PR TITLE
Adding a PR template

### PR DESCRIPTION
**Pull request by vogella**
This will make the PR more informative, example:

image

### CODE REVIEW

**Comment by laeubi** [comment]
@vogella can you check if it is possible to put the template on the organization level?

File: .github/workflows/create-pullrequest.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Create or update pull request for non-master branches
```
Comment (laeubi) [inline .github/workflows/create-pullrequest.yml:1]: Can you explain what this action does? This is not quite clear to me...

Comment (vogella) [reply .github/workflows/create-pullrequest.yml:1]: > Can you explain what this action does? This is not quite clear to me...

See here https://github.com/devops-infra/action-pull-request

Comment (laeubi) [reply .github/workflows/create-pullrequest.yml:1]: I must confess I don't understand this ... why is standard github Pullrequest not enough? This seems doing much more than "Adding a PR template" I certainly don't want that each and every branch is automatically created as a PR ... also all information added is already visible on the "Files changed" Tab already.

**Comment by laeubi** [comment]
For example look at
- https://github.com/eclipse-jdt/.github/blob/main/pull_request_template.md
- https://github.com/eclipse-platform/eclipse.platform.swt/tree/master/.github/ISSUE_TEMPLATE

that seems much more useful than this (from what I understand from the short description given).

**Pull request closed without merging**